dist/templates/login.eta

```diff
@@ -1,24 +1,25 @@
 <% layout('./layout') %>
-<p>Hi <%= it.member.name %>,</p>
-<p>Sometimes you just want to send a simple HTML email with a simple design and clear call to action. This is it.</p>
-<% if (it.reRegistrationAttempt) { %>
-  <p>There was a re-registration attempt using you email. If you did this by mistake, no problem. Here's the link to login.</p>
-  <p>If it wasn't you, don't worry you account and data are safe.</p>
-<% } %>
-<table role="presentation" border="0" cellpadding="0" cellspacing="0" class="btn btn-primary">
-  <tbody>
-    <tr>
-      <td align="left">
-        <table role="presentation" border="0" cellpadding="0" cellspacing="0">
-          <tbody>
-            <tr>
-              <td> <a href="<%= it.link %>" target="_blank">Sign In</a> </td>
-            </tr>
-          </tbody>
-        </table>
-      </td>
-    </tr>
-  </tbody>
-</table>
-<p>This is a really simple email template. Its sole purpose is to get the recipient to click the button with no distractions.</p>
-<p>Good luck! Hope it works.</p>
+  <h2><b>Hello!</b></h2>
+
+  <p>You asked us to send you a magic link for quickly signing in to your account. Here your go :</p>
```

Comment:
How are we going to handle multiple languages? Usually this can be injected from a JSON.

Reply:
Typo: `"Here you go:"`